plugins/runekit

```diff
@@ -1,3 +1,3 @@
 repository=https://github.com/ImToxicCode/runekit.git
-commit=3ae3e07e2bbe0381166c3508c96bd09373f364ac
-warning=This plugin communicates the stage of crops and birdhouses to a 3rd party server not controlled or verified by the RuneLite Developers.
+commit=caedbaafef4b1b00fe36f1b123fb06dd26572b42
+warning=This plugin sends data to a 3rd party server not controlled or verified by the RuneLite Developers.
```

Comment:
Maybe this can be a little more specific, eg.
```suggestion
warning=This plugin sends player data to a 3rd party server not controlled or verified by the RuneLite Developers.
```